### ISSUE TITLE
Crash in ArgoCD

### ISSUE THREAD

**Issue by greenkiwi** [argoproj-labs/rollout-extension#10]
When I click the `more` tab on a rollout, I get the following stack trace.

Browser: `Chrome Mac OS`
ArgoCD: `v2.3.3+07ac038`
Argo Rollouts: `v1.2.0+08cf10e`
ArgoRolloutExtension: `main` branch

```
Something went wrong!

Consider submitting an issue [here](https://github.com/argoproj/argo-cd/issues/new?labels=bug&template=bug_report.md).


Stacktrace:

TypeError: Cannot read properties of undefined (reading 'name')
    at anchor (https://argocd.core-services.r.m/main.c7ea22e999b3805bc676.js:54:342605)
    at qo (https://argocd.core-services.r.m/main.c7ea22e999b3805bc676.js:404:57924)
    at ga (https://argocd.core-services.r.m/main.c7ea22e999b3805bc676.js:404:104163)
    at cl (https://argocd.core-services.r.m/main.c7ea22e999b3805bc676.js:404:96711)
    at al (https://argocd.core-services.r.m/main.c7ea22e999b3805bc676.js:404:96636)
    at Qa (https://argocd.core-services.r.m/main.c7ea22e999b3805bc676.js:404:93666)
    at https://argocd.core-services.rr.mu/main.c7ea22e999b3805bc676.js:404:45314
    at t.unstable_runWithPriority (https://argocd.core-services.r.m/main.c7ea22e999b3805bc676.js:412:3844)
    at zr (https://argocd.core-services.r.m/main.c7ea22e999b3805bc676.js:404:45023)
    at Gr (https://argocd.core-services.r.m/main.c7ea22e999b3805bc676.js:404:45259)
```

**Comment by alexef** [comment]
This is the same issue as https://github.com/argoproj-labs/rollout-extension/issues/2, solved in #18 .

Can we get a new release?

**Comment by alexef** [comment]
actually this is not the same issue. using the debugger I found this is actually caused by: https://github.com/argoproj-labs/rollout-extension/pull/38
